tensorflow/compiler/xla/service/gpu/gemm_rewriter.cc

```diff
@@ -501,9 +501,50 @@ class GemmRewriterVisitor : public DfsHloRewriteVisitor {
                       CublasLtMatmulMaybeF8(&existing_gemm).WithOneUser())
                       .WithOneUser(),
                   m::Broadcast(&bias, m::Op())))) {
-      TF_ASSIGN_OR_RETURN(
-          bool was_fused,
-          FuseVectorBiasAdd(instr, bias, existing_gemm, optional_slice));
+      HloInstruction *optional_bias_f16 = nullptr;
+      if (existing_gemm->custom_call_target() == kCublasLtMatmulF8CallTarget &&
```

Comment:
You should check `CUDA_VERSION > 12000` somewhere.

Reply:
Vector bias is always supported in cuda <=12. Matrix bias is not .

Reply:
Oh oops I got mixed up with your other PR https://github.com/tensorflow/tensorflow/pull/60319. And same for the comment "Do you still need the logic to handle the matrix bias in CreateF8CustomCall?", which I already asked in the other PR